### DISCUSSION TITLE
Does OpenRadioss can perform a powder-forming compaction simulation?

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2595] [Users]
Dear researchers,
I hope you are doing well.

I need help with a simulation using OpenRadioss software. Specifically, I want to simulate powder-forming compaction by importing particles (e.g. 400 particles) into a box and applying pressure with a punch to compact them. I need to define a general contact between the particles and the box walls as well as between the particles themselves. (Please see the attached figure). Can you please let me know if this software can handle this type of simulation? If not, do you know of any open-source software that can do this? Thanks!
![1](https://github.com/OpenRadioss/OpenRadioss/assets/169218266/49d3381b-e7ed-4742-9a1e-a752dd9420a1)

**Comment by MarianBulla** [comment]
Dear @Majid-MHZ,
thank you very much for your intereste in OpenRadioss.
Yes, I think it should be doable with OpenRadioss.
There are severall possibilites, how to realize it.
The first approach would be to try it using SPH particles (Smooth Particles Hydrodynamics).
Contacts to surounding structure, which can be also deformable, is possible.
If you already have a mesh or geometry, in case of only 400 particles, one can go with a full geometrical analysis. Just use a suited unit for your dimension of the length (millimeters or nanometers or ...).
Hope it helps you ?

Thanks and best regards,
Marian

**Reply by Majid-MHZ** [reply]
Dear @MarianBulla ,
Thank you for reply. 
I am doing a powder-forming compaction (multi-particles) simulation by Abaqus, and I have been requested to perform this simulation in an open-source FEM. I found OpenRadioss a great open-source FEM, But I am literally new to this software. I have some questions for you. The questions are as follows:

Is it possible to import an Abaqus .inp file to this software? 
How can I import my assembly model to this software? 
Do you have any tutorial in this regard?

Best regards,
Majid

**Comment by rzadki22** [comment]
Hello @Majid-MHZ ,

your case looks as a very interesting challenge! Do you usually use Abaqus Explicit or Implicit solver for this case?
As far as I know there is no available better software for such high nonlinear case than OpenRadioss.  Nevertheless if you use Abaqus than you can still try Calculix.

It is not possible to read .inp file inside Radioss, you must convert your mesh with a preprocessor.

Can you describe something more about thermal condition during analysis?

Best regards,
Filip

**Reply by elequiniou** [reply]
Indeed, this is now possible to read .inp from the OpenRadioss GUI, please, have a look at this announcement: https://github.com/orgs/OpenRadioss/discussions/3222 
Hope it helps and look forward to your feedback!
Kind Regards,
Eric

**Comment by Mishal-Alpine** [comment]
Hello @Majid-MHZ 

You could also look into the use of LIGGGHTS for your application.
